.github/workflows/es-spellcheck.yml

```diff
@@ -13,6 +13,7 @@ name: Spellcheck CI
 on:
   pull_request:
     paths:
+      - 'content/es/**/*.md'
       - 'content/es/*.md'
```

Comment:
Is the rule `content/es/**/*.md` includes the rule `content/es/*.md`?
If so, then the rule `content/es/*.md` seems okay to be removed.

(In terms of consistency with L14 of `content/es/.spellcheck.yml`)

Reply:
Well, I'm not 100% sure if the same logic applies here given they're different tools, but I can give it a try.